extension.js

```diff
@@ -82,7 +83,7 @@ const _display_handles = [];
 
 function enable() {
   _display_handles.push(global.display.connect('window-created', (_, win) => {
-    global.run_at_leisure(checkFullScreen.bind(this, win));
+    GLib.idle_add(GLib.PRIORITY_IDLE, checkFullScreen.bind(this, win));
```

Comment:
The documentation states that https://gjs-docs.gnome.org/glib20~2.62.0-constants/ the constant is either PRIORITY_DEFAULT or PRIORITY_DEFAULT_IDLE . This will not work saying unknown property.

This does not seem to work, event with PRIORITY_LOW. I faced this problem earlier, the reason I added run at leisure was because the window seemed to move to new workspace but was never drawn/rendered. So basically you are left with a window that you can interact with through keyboard and mouse but is never drawn so you can never see what you are interacting with and how. Window does show up in workspace preview when pressing super key. The only way to get it drawn is to minimize and unminimize the moved window.

The above behaviour is on my Ubuntu 19.10, if this has been fixed upstream in a newer version of gnome, I do not know about it. The extension is primarily targeted at the system I am currently using. For now the system is using Ubuntu 19.10 but will soon be 20.04 LTS, once it is released, for which I intend to support this extension. 

Unless the fixes or workarounds are back-ported for this issue, I think I would keep the current run_at_leisure for now as it is a good workaround for this issue and the extension behaves as expected.

I have fixed the DnD problem at https://github.com/raonetwo/MaximizeToWorkspace/commit/b916d5c3f32d3e4f290ddf1aa3e49b40062ae398 where I am checking frame and layer of the window. 

```suggestion
    GLib.idle_add(GLib.PRIORITY_LOW, checkFullScreen.bind(this, win));
```